hepcrawl/spiders/aps_spider.py

```diff
@@ -55,9 +54,10 @@ class APSSpider(LastRunStoreSpider):
 
     @strict_kwargs
     def __init__(self, from_date=None, until_date=None,
-                 date="published", sets=None, per_page=100,
+                 date="published", sets=None, per_page=100, aps_url=None,
```

Comment:
is this for easier testing?

Reply:
Yes. Also I hate hard-coded URLs which you cannot change easily.